source/preface.tex

```diff
@@ -1,4 +1,19 @@
 %!TEX root = std.tex
+
 \chapter{Foreword}
 
 [This page is intentionally left blank.]
+
+\rSec0[introduction]{Introduction}
+
+Clauses and subclauses in this document are annotated
+with a so-called stable name,
+presented in square brackets next to the (sub)clause heading
+(such as ``[introduction]'' for this clause).
```

Comment:
With the above, the introduction would no longer be a Clause.

Reply:
Are unnumbered things not "Clauses"? That's news to me.

Reply:
["22.3	Numbering, subdivision and hanging paragraphs"](https://www.iso.org/sites/directives/current/part2/index.xhtml#_idTextAnchor314) says
> 22.3.1	Numbering
The clauses in each document or part shall be numbered with Arabic numerals, starting with 1 for the Scope (see Figure 3).

Reply:
I think it's unclear whether Foreword and Introduction are clauses, though an Introduction can have subclauses. And they're definitely not numbered.

Reply:
My two quotes in this PR so far say "clauses are numbered" and "the introduction is unnumbered".